packages/angular_devkit/core/src/json/schema/registry.ts

```diff
@@ -119,7 +121,6 @@ export class CoreSchemaRegistry implements SchemaRegistry {
     }
 
     this._ajv = ajv({
-      removeAdditional: 'all',
```

Comment:
Setting it to any other option but false also causes the validation to succeed instead of failing. Specifically, the `fails on invalid additionalProperties` test in `packages\angular_devkit\core\src\json\schema\registry_spec.ts` starts failing.